### PR TITLE
Friends table

### PR DESCRIPTION
**Pull request by Stearnzy**
Friendship table made
Friendships and Users associated

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,12 @@
...
 #   Character.create(name: 'Luke', movie: movies.first)
+
+
+user1 = User.create(first_name: 'Zach', last_name: 'Stearns', email: 'zach@email.com', password: 'password')
+user2 = User.create(email: 'test@gmail.com', password: 'test', first_name: 'Alex', last_name: 'Rivero')
+
+
+user4 = User.create(first_name: 'Zach', last_name: 'Stearns', email: 'zach12@email.com', password: 'password')
+
+friends = Friendship.create!(user_id: 4, friend_id: 3)
```
Comment (Eternal-Flame085) [inline db/seeds.rb:16]: Good making seeds to have some data

File: app/models/friendship.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Friendship < ApplicationRecord
+  belongs_to :user
+  belongs_to :friend, class_name: 'User'
+end
```
Comment (Eternal-Flame085) [inline app/models/friendship.rb:4]: Nice, figuring out how to get this to work

**Review by stillsheryl — APPROVED** [review]
Glad we finally got this working! And it's more straight forward than I had imagined. :)

File: app/models/user.rb [hunk 3]
```diff
@@ -1,9 +1,12 @@
 class User < ApplicationRecord
   has_secure_password
-  validates_presence_of :email, :first_name, :last_name
-  validates_uniqueness_of :email
-  validates_confirmation_of :password
+  validates :email, :first_name, :last_name, presence: true
+  validates :email, uniqueness: true
+  validates :password, confirmation: true
```
Comment (Eternal-Flame085) [inline app/models/user.rb:5]: good changes

**Review by Eternal-Flame085 — APPROVED** [review]

**Pull request merged**